latest cloudflare quickstart & how-to updates (#309)
auth0/docs-v2: auth4genai/snippets/common/configure-cloudflare-kv-store.mdx

> For persisting Auth0 session data and other key-value pairs, you'll want to configure a persistent store with your Cloudflare agent worker. One easy methodology is using Workers KV and a [KV namespace](https://developers.cloudflare.com/kv/get-started/) with your Cloudflare agent workers. This will enable you to store Auth0 session data and other key-value pairs with easy access from your Cloudflare agent workers. This can be used when constructing the `CloudflareKVStore` instance with your Cloudflare agent worker.

Reviewer: ```suggestion
For persisting Auth0 session data and other key-value pairs, you need to configure a persistent store with your Cloudflare agent worker. When constructing the `CloudflareKVStore` instance with your Cloudflare agent worker, you can use Workers KV and a [KV namespace](https://developers.cloudflare.com/kv/get-started/) as the persistent store. This enables you to store Auth0 session data and other key-value pairs with easy access from your Cloudflare agent workers.
```

```ts
import { CloudflareKVStore } from '@auth0/ai-cloudflare';
...

return new CloudflareKVStore({ kv: this.env.YOUR_KV_NAMESPACE });
```

> **Note:** the `kv` prop accepts any store which implements the [KVNamespace interface](https://github.com/auth0/auth0-ai-js/blob/%40auth0/ai-cloudflare-v2.0.0/packages/ai-cloudflare/src/CloudflareKVStore.ts#L3-L14), so any persistent store which implements this interface will work.

Reviewer: ```suggestion
**Note:** the `kv` prop accepts any store which implements the [`KVNamespace` interface](https://github.com/auth0/auth0-ai-js/blob/%40auth0/ai-cloudflare-v2.0.0/packages/ai-cloudflare/src/CloudflareKVStore.ts#L3-L14), so any persistent store which implements this interface will work.
```
